signer/src/stacks/api.rs

```diff
@@ -1574,7 +1504,84 @@ fn extract_signatures_required(value: Value) -> Result<Option<u16>, Error> {
     }
 }
 
+#[derive(Debug, Deserialize)]
+struct GetTenureHeadersLightApiResponse {
```

Comment:
nit: we can shorten the name to `GetTenureHeadersResponse`, I think this is partially a left over name of the earlier implementation.